content-src/components/DiscoveryStreamComponents/DSDismiss/DSDismiss.jsx

```diff
@@ -70,7 +70,9 @@ export class DSDismiss extends React.PureComponent {
           onMouseEnter={this.onHover}
           onMouseLeave={this.offHover}
         >
-          <span className="icon icon-dismiss" />
+          <span className="icon icon-dismiss">
+            <span className="sr-only">Dismiss</span>
```

Comment:
What does sr-only mean?

Reply:
I added a utility class+mixin (used Bootstrap's approach) for hiding text content offscreen so that only screenreaders can pick it up.

Reply:
This seems to be re-adding an anti-pattern that was explicitly removed for accessibility reasons https://github.com/mozilla/activity-stream/commit/b2a197b8c4a772d4642f5f1b08a0a40a5b1d16a3

Reply:
Yeah I don't think I knew the whole story here, but if sr-only is an issue we should figure out how to get this done the right way.